### PR TITLE
Cleaned up Spanish translation of Java While loop guide

### PR DESCRIPTION
**Pull request by lyurrutia**
- [ x ] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [ x ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [ x ] My pull request targets the `master` branch of freeCodeCamp.
- [ x ] None of my changes are plagiarized from another source without proper attribution.
- [ x ] My article does not contain shortened URLs or affiliate links.

### CODE REVIEW

**Comment by RandellDawson** [comment]
Closed and Reopened this PR to attempt to resolve a specific Travis build failure.

File: guide/spanish/java/loops/while-loop/index.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ localeTitle: Mientras bucle
 ---
 # Mientras bucle
 
-El `while` bucle se ejecuta repetidamente el bloque de instrucciones hasta que la condición especificada dentro de los paréntesis se evalúa como `false` . Por ejemplo:
+El `while` bucle repetidamente ejecuta el bloque de instrucciónes hasta que la condición especificada dentro de los paréntesis se evalúa como `false` . Por ejemplo:
```
Comment (hestellez) [inline guide/spanish/java/loops/while-loop/index.md]: - Debería ser **El bucle `while`** 
- No se necesita el espacio después de `false`

File: guide/spanish/java/loops/while-loop/index.md [hunk 2]
```diff
@@ -13,13 +13,12 @@ while (some_condition_is_true)
  } 
 ```
 
-Cada 'iteración' (de ejecutar el bloque de sentencias) está precedida por la evaluación de la condición especificada entre paréntesis: las sentencias se ejecutan solo si la condición se evalúa como `true` . Si se evalúa como `false` , la ejecución del programa se reanuda desde la instrucción justo después del bloque `while` .
+Cada 'iteración' (de ejecutar el bloque de instrucciónes) está precedida por la evaluación de la condición especificada dentro del paréntesis: las instrucciónes se ejecutan solo si la condición se evalúa como `true` . Si se evalúa como `false` , la ejecución del programa se reanuda desde la instrucción justo después del bloque `while`.
```
Comment (hestellez) [inline guide/spanish/java/loops/while-loop/index.md]: - instrucciones no lleva tilde
- es mejor **dentro**, o en otro caso, **dentro de los paréntesis**...

File: guide/spanish/java/loops/while-loop/index.md [hunk 3]
```diff
@@ -13,13 +13,12 @@ while (some_condition_is_true)
...
 
-**Nota:** Para el `while` de bucle para iniciar la ejecución, que le requiere la condición de ser `true` inicialmente. Sin embargo, para salir del bucle, debe hacer algo dentro del bloque de sentencias para llegar a una iteración cuando la condición se evalúe como `false` (como se hace a continuación). De lo contrario el bucle se ejecutará para siempre. (En la práctica, se ejecutará hasta que la [JVM](https://guide.freecodecamp.org/java/the-java-virtual-machine-jvm) se quede sin memoria).
+**Nota:** Para que el `while` bucle empieze la ejecución, requiere que la condición sea `true` inicialmente. Sin embargo, para salir del bucle, debe hacer algo dentro del bloque de instrucciónes para llegar a una iteración cuando la condición se evalúe como `false` (como en el siguiente ejemplo). De lo contrario, el bucle se ejecutará para siempre. (En realidad, se ejecutará hasta que la [JVM](https://guide.freecodecamp.org/java/the-java-virtual-machine-jvm) se quede sin memoria).
```
Comment (hestellez) [inline guide/spanish/java/loops/while-loop/index.md]: - **empiece**
- instrucciones no lleva tilde

**Review by hestellez — CHANGES_REQUESTED** [review]

**Comment by RandellDawson** [comment]
@murrutiar Can you please make the suggested changes @hestellez has suggested.

Thank you.

**Pull request closed without merging**